### PR TITLE
Implement guess command

### PR DESCRIPTION
**Pull request by uu59**

### CODE REVIEW

File: lib/embulk/input/mixpanel.rb [hunk 1]
```diff
@@ -27,15 +29,22 @@ def self.resume(task, columns, count, &control)
...
+        properties = Guess::SchemaGuess.from_hash_records(sample_records.map{|r| r["properties"]})
+        columns = properties.map do |col|
+          result = {
+            name: col.name,
+            type: col.type,
+          }
+          result[:format] = col.format if col.format
+          result
+        end
+        columns.unshift({name: "event", type: :string})
```
Comment (yoshihara) [inline lib/embulk/input/mixpanel.rb]: :eyes: `{}` is needless

File: lib/embulk/input/mixpanel_api/client.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          @api_secret = api_secret
+        end
+
+        def export(params = {})
+          # https://mixpanel.com/docs/api-documentation/exporting-raw-data-you-inserted-into-mixpanel
+          params[:api_key] = api_key
+          params[:expire] ||= Time.now.to_i + TIMEOUT_SECONDS
+          params[:sig] = signature(params)
+          response = httpclient.get(ENDPOINT_EXPORT, params)
+          Enumerator.new do |y|
```
Comment (yoshihara) [inline lib/embulk/input/mixpanel_api/client.rb:27]: I want to add blank line above this line... :pray:

File: test/embulk/input/test_mixpanel.rb [hunk 3]
```diff
@@ -1,11 +1,63 @@
 require "prepare_embulk"
+require "embulk/input/mixpanel_api/client"
```
Comment (yoshihara) [inline test/embulk/input/test_mixpanel.rb]: This require is needed? (Already required by "embulk/input/mixpanel_api/client")

File: lib/embulk/input/mixpanel.rb (hunk 1 above)
Comment (uu59) [inline lib/embulk/input/mixpanel.rb]: Removed

File: lib/embulk/input/mixpanel_api/client.rb (hunk 2 above)
Comment (uu59) [inline lib/embulk/input/mixpanel_api/client.rb:27]: Added :white_check_mark:

File: test/embulk/input/test_mixpanel.rb (hunk 3 above)
Comment (uu59) [inline test/embulk/input/test_mixpanel.rb]: Removed

**Comment by yoshihara** [comment]
LGTM

**Pull request merged**